### PR TITLE
chore: remove gulp task to update Homebrew Formula

### PR DESCRIPTION
**Pull request by 10ko**
**What this PR does / why we need it**:
Since the we tested the github action to update the Homebrew Formula succesfully, we can now safely remove the old `gulp` task we used to do that.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -230,11 +230,8 @@ To make a new release, set your current working directory to the garden root dir
     * To get a list of all contributors between releases, ordered by count, run: `git log <previous-tag>..<current-tag> --no-merges "$@" | grep ^Author | sort | uniq -c | sort -nr`. Note that authors of squashed commits won't show up so it might be good to do a quick sanity check on Github as well.
 7. Click the **Publish release** button.
 8. Make a pull request for the branch that was pushed by the script.
-9. **Update the Homebrew package: `gulp update-brew`.**
-10. Install the Homebrew package and make sure it works okay:
```
Comment (edvald) [inline CONTRIBUTING.md]: I think I'd like to keep this one around.

File: gulpfile.ts [hunk 2]
```diff
@@ -6,21 +6,12 @@
...
  */
 
-import { resolve } from "path"
-import execa from "execa"
-import { writeFile, readFile, ensureDir, pathExists, remove } from "fs-extra"
-import { getUrlChecksum } from "./support/support-util"
-import handlebars from "handlebars"
-import { find } from "lodash"
-
-const Octokit = require("@octokit/rest")
```
Comment (edvald) [inline gulpfile.ts]: We should remove packages we don't need anymore from the package.json.

File: CONTRIBUTING.md (hunk 1 above)
Comment (10ko) [reply CONTRIBUTING.md]: Ok but FYI the github action runs on mac and installs/runs `garden --version`.

File: gulpfile.ts (hunk 2 above)
Comment (10ko) [reply gulpfile.ts]: Ops, I would have sweared we use Oktokit somewhere else.
Removed

Comment (10ko) [reply gulpfile.ts]: Ops, I would have sweared we use Oktokit somewhere else.
Removed

Comment (eysi09) [reply gulpfile.ts]: We could try this: https://www.npmjs.com/package/depcheck

**Pull request merged**